src/main/cpp/t05_minodd.cpp

```diff
@@ -29,4 +29,22 @@ using namespace std;
 
 int t05_minodd() {
 
+    int N ;
+    int max = 0 ;
+    int v[100] ;
+    cin >> N ;
+    for ( int i = 0; i < N; ++i ) {
+        cin >> v[i];
+    }
+    for ( int i = 0 ; i < N ; i++ ) {
+        if ( v[i] % 2 != 0 && v[i] % 2 != 0 && max == 0 )
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Same expression on both sides of '&&'.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4732197)